Tools/build/consts_getter.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+ROOT = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+INTERNAL = os.path.join(ROOT, 'Include', 'internal')
```

Comment:
I guess we could rewrite that as follows:
```suggestion
from pathlib import Path

ROOT = Path(__file__).resolve().parents[2]
INTERNAL = ROOT / "Include" / "internal"
```